lib/y2_module_basetest.pm

```diff
@@ -87,6 +88,22 @@ sub accept_warning_network_manager_default {
     assert_screen 'yast2_lan-global-tab';
 }
 
+
+=head2 wait_for_exit
+
+ wait_for_exit($module);
+
+Wait for string yast2-<module>-status-0 (which has been
+previously used to open the module) to appear in the serial output
+in order to ensure that the module exited.
+
+=cut
+sub wait_for_exit {
+    my $module = shift;
+    wait_serial("yast2-$module-status-0") ||
```

Comment:
I believe that we had some cases where default timeout was to small, in the code we have it equal to 180 seconds and even 400 for the smt. So we at least need a parameter here, not sure about good default, we need to check current values.